doc/api/worker_threads.md

```diff
@@ -1191,9 +1191,16 @@ changes:
       used for generated code.
     * `stackSizeMb` {number} The default maximum stack size for the thread.
       Small values may lead to unusable Worker instances. **Default:** `4`.
-  * `name` {string} An optional `name` to be appended to the worker title
-    for debugging/identification purposes, making the final title as
-    `[worker ${id}] ${name}`. **Default:** `''`.
+  * `name` {string} An optional `name` to be replaced in the thread name
+    and to the worker title for debugging/identification purposes,
+    making the final title as `[worker ${id}] ${name}`.
+    This parameter has a maximum allowed size, depending on the operating
+    system. If the provided name exceeds the limit, it will be truncated
+    * Maximum sizes:
+      * Windows: 32,767 characters
+      * macOS: 64 characters
+      * Other systems: 16 characters
+        **Default:** `'WorkerThread'`.
```

Comment:
Here are the limit that libuv uses: https://github.com/libuv/libuv/commit/61c966cf0b4b6b5fc44d5ab3678cf5d0caef249d#diff-633663e593ffc06d66dce99513aefc6db4b74ead8f760914d1e882e196bae225R431-R443